doc/source/user_guide/timeseries.rst

```diff
@@ -905,11 +905,11 @@ into ``freq`` keyword arguments. The available date offsets and associated frequ
     :class:`~pandas.tseries.offsets.CustomBusinessHour`, ``'CBH'``, "custom business hour"
     :class:`~pandas.tseries.offsets.Day`, ``'D'``, "one absolute day"
     :class:`~pandas.tseries.offsets.Hour`, ``'H'``, "one hour"
```

Comment:
Thank you for the comments. I agree, it would be better to rename 'H', 'CBH', and 'BH' too, I'll do it in a separate PR. I like the idea to keep anything sub-daily lowercase, and daily or higher upper-case.